libraries/lib-xml/XMLMethodRegistry.h

```diff
@@ -42,10 +43,11 @@ using Mutators = std::vector< std::pair< std::string, Mutator<Substructure> > >;
 protected:
    using TypeErasedObjectAccessor = std::function< XMLTagHandler *( void* ) >;
    using TagTable =
-      std::unordered_map< std::string, TypeErasedObjectAccessor >;
+      std::unordered_map< std::string_view, TypeErasedObjectAccessor >;
    TagTable mTagTable;
+   std::forward_list<std::string> mTags;
```

Comment:
Probably `std::deque`? It can be more efficient on some platforms

Reply:
I thought `deque` deals with big pages.  Existing ProjectFileIORegistry (alias XMLMethodRegistry<AudacityProject>) registrations are maybe too few to justify it, and they happen only at startup.

Whereas, the lookups into the map may be many, and we want all the savings we can find during deserialization.

The lookups may not really be many, yet, but XMLMethodRegistry may be reused for many more fields later.